aio/content/navigation.json

```
@@ -102,6 +102,11 @@
       "title": "Understanding Angular",
       "tooltip": "Learn the concepts essential to becoming a proficient Angular developer.",
       "children": [
+        {
+          "title": "Overview",
+          "url": "guide/understanding-angular-overview",
+          "tooltip": "Learn about the most common elements of Angular applications."
```

```suggestion
          "tooltip": "Learn about the most common parts of an Angular application"
```

The `elements` term refers to the tag level structure in html or the DOM.  Is there a better term that fits? Maybe `part`.

Changed phrase to single target.